magicauth/next_url.py

```diff
@@ -24,9 +24,7 @@ def get_next_url(self, request):
         next_url = request.GET.get("next")
         if not next_url:
             next_url = reverse(magicauth_settings.LOGGED_IN_REDIRECT_URL_NAME)
-        # the following `is_safe_url` will be deprecated in django 4 and replaced by
-        # url_has_allowed_host_and_scheme
-        if not is_safe_url(next_url, allowed_hosts={request.get_host()}, require_https=True):
+        if not url_has_allowed_host_and_scheme(next_url, allowed_hosts={request.get_host()}, require_https=True):
```

Comment:
It's more secure to put the hostname in an env var, rather that request.get_host(), because the host header can get spoofed. 
Source : https://help.semmle.com/wiki/display/JS/Host+header+poisoning+in+email+generation

Reply:
resolved in #38